packaging/installer/install-required-packages.sh

```diff
@@ -129,1282 +129,1223 @@ EOF
+  fi
+
+  # -------------------------------------------------------------------------
+  # applications needed for the netdata demo sites
+
+  if [ "${PACKAGES_NETDATA_DEMO_SITE}" -ne 0 ]; then
+    require_cmd sudo || suitable_package sudo
+    require_cmd jq || suitable_package jq
+    require_cmd nginx || suitable_package nginx
+    require_cmd postconf || suitable_package postfix
+    require_cmd lxc-create || suitable_package lxc
+    require_cmd logwatch || suitable_package logwatch
+    require_cmd mail || suitable_package mailutils
+    require_cmd iostat || suitable_package sysstat
+    require_cmd iotop || suitable_package iotop
+  fi
 }
 
 DRYRUN=0
 run() {
 
-	printf >&2 "%q " "${@}"
-	printf >&2 "\n"
+  printf >&2 "%q " "${@}"
+  printf >&2 "\n"
 
-	if [ ! "${DRYRUN}" -eq 1 ]
-		then
-		"${@}"
-		return $?
-	fi
-	return 0
+  if [ ! "${DRYRUN}" -eq 1 ]; then
+    "${@}"
+    return $?
+  fi
+  return 0
 }
 
 sudo=
-if [ ${UID} -ne 0 ]
-	then
-	sudo="sudo"
+if [ ${UID} -ne 0 ]; then
+  sudo="sudo"
 fi
 
 # -----------------------------------------------------------------------------
 # debian / ubuntu
 
 validate_install_apt_get() {
-	echo >&2 " > Checking if package '${*}' is installed..."
-	[ "$(dpkg-query -W --showformat='${Status}\n' "${*}")" = "install ok installed" ] || echo "${*}"
+  echo >&2 " > Checking if package '${*}' is installed..."
+  [ "$(dpkg-query -W --showformat='${Status}\n' "${*}")" = "install ok installed" ] || echo "${*}"
 }
 
 install_apt_get() {
-	# download the latest package info
-	if [ "${DRYRUN}" -eq 1 ]
-		then
-		echo >&2 " >> IMPORTANT << "
-		echo >&2 "    Please make sure your system is up to date"
-		echo >&2 "    by running:  ${sudo} apt-get update  "
-		echo >&2 
-	fi
-
-	local opts=""
-	if [ ${NON_INTERACTIVE} -eq 1 ]
-		then
-		echo >&2 "Running in non-interactive mode"
-		# http://serverfault.com/questions/227190/how-do-i-ask-apt-get-to-skip-any-interactive-post-install-configuration-steps
-		export DEBIAN_FRONTEND="noninteractive"
-		opts="${opts} -yq"
-	fi
-
-	# install the required packages
-	for pkg in "${@}"; do
-		[[ ${DRYRUN} -eq 0 ]] && echo >&2 "Adding package ${pkg}"
-		run ${sudo} apt-get ${opts} install "${pkg}"
-	done
+  # download the latest package info
+  if [ "${DRYRUN}" -eq 1 ]; then
+    echo >&2 " >> IMPORTANT << "
+    echo >&2 "    Please make sure your system is up to date"
+    echo >&2 "    by running:  ${sudo} apt-get update  "
+    echo >&2
+  fi
+
+  local opts=""
+  if [ "${NON_INTERACTIVE}" -eq 1 ]; then
+    echo >&2 "Running in non-interactive mode"
+    # http://serverfault.com/questions/227190/how-do-i-ask-apt-get-to-skip-any-interactive-post-install-configuration-steps
+    export DEBIAN_FRONTEND="noninteractive"
+    opts="${opts} -yq"
+  fi
+
+  # install the required packages
+  for pkg in "${@}"; do
+    [[ ${DRYRUN} -eq 0 ]] && echo >&2 "Adding package ${pkg}"
+    run ${sudo} apt-get "${opts}" install "${pkg}"
```

Comment:
I think this should be:
```suggestion
    run ${sudo} apt-get ${opts} install "${pkg}"
```

Reply:
Ahh actually according to [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086) we should use an array expansion here `${foo[@]}` at least in Bash.